api/deployments/component_handler.go

```diff
@@ -96,14 +97,22 @@ func (deploy *deployHandler) getComponent(ctx context.Context, component v1.Radi
 	return deploymentComponent, nil
 }
 
-func (deploy *deployHandler) getHpaSummary(ctx context.Context, component v1.RadixCommonDeployComponent, envNs string) (*deploymentModels.HorizontalScalingSummary, error) {
-	hpa, err := deploy.accounts.UserAccount.Client.AutoscalingV2().HorizontalPodAutoscalers(envNs).Get(ctx, component.GetName(), metav1.GetOptions{})
+func (deploy *deployHandler) getHpaSummary(ctx context.Context, component v1.RadixCommonDeployComponent, appName, envNs string) (*deploymentModels.HorizontalScalingSummary, error) {
+	selector := labelselector.ForComponent(appName, component.GetName()).String()
+	hpas, err := deploy.accounts.UserAccount.Client.AutoscalingV2().HorizontalPodAutoscalers(envNs).List(ctx, metav1.ListOptions{LabelSelector: selector})
```

Comment:
I don't think List() can return an IsNotFound error, so you can probably simplify the if block below to return if err!=nil. No need to check for errors.IsNotFound.

Reply:
Nice, fixed